DQM/L1TMonitor/src/L1TMP7ZeroSupp.cc

```diff
@@ -231,16 +231,16 @@ void L1TMP7ZeroSupp::analyze(const edm::Event& e, const edm::EventSetup& c) {
         continue;
 
       auto payload64 = amc.data();
-      auto start = (const uint32_t*)payload64.get();
+      auto start = reinterpret_cast<const uint32_t*>(&payload64.front());
       // Want to have payload size in 32 bit words, but AMC measures
       // it in 64 bit words -> factor 2.
-      const uint32_t* end = start + (amc.size() * 2);
+      const uint32_t* end = start + (payload64.size() * 2);
 
       auto payload = std::make_unique<l1t::MP7Payload>(start, end, false);
 
       // getBlock() returns a non-null unique_ptr on success
```

Comment:
Unless I'm mistaken I think it would be good to update (or remove) this comment.